src/site/content/en/how-to-optimize-inp/diagnose-slow-interactions-in-the-lab/index.md

```diff
@@ -211,7 +211,7 @@ The [rendering performance guide](/rendering-performance/) is a good primer on u
 {% endAside %}
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/Y9mNXlHmq8sOvQgVgv5Y.png", alt="Rendering work as visualized in the performance panel of Chrome DevTools. The rendering work occurs both within an event callback, as well as after the event callback in order to paint the next frame.", width="730", height="1140" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pFmaZMXrlbugrC9WPoHM.png", alt="Rendering work as visualized in the performance panel of Chrome DevTools. The rendering work occurs both within an event callback, as well as after the event callback in order to paint the next frame.", width="800", height="229" %}
```

Comment:
It no longer seems to happen in the event callback in the new example. So either change the example or update the Alt text.